modules/serverless-logic-security-config-auth-credentials-openapi.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+// serverles-logic/serverless-logic-authentication-openapi-services
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-security-config-auth-credentials-openapi_{context}"]
+= Configuring authentication credentials for OpenAPI services
+
+To invoke OpenAPI service operations secured by authentication schemes, you must configure the corresponding credentials and parameters in your application. {ServerlessLogicProductName} uses these configurations to authenticate with the external services during workflow execution.
+
+This procedure describes how to define and apply the necessary configuration properties for security schemes declared in the OpenAPI specification file. You can use either `application.properties`, the `ConfigMap` associated to your workflow, or environment variables to provide these credentials.
+
+[NOTE]
+====
+The security schemes defined in an OpenAPI specification file are global to all the operations that are available in the same file. This means that the configurations set for a particular security scheme also apply to the other secured operations.
+====
+
+.Prerequisites
+
+* You have access to a {ServerlessLogicProductName} project with the appropriate roles and permissions to create applications and other workloads in {ocp-product-title}.
+* Your OpenAPI specification is defined in one or more security schemes.
+* You have access to the OpenAPI specification files.
+* You have identified the schemes you want to configure `http-basic-example` or `api-key-example`.
+* You have access to the application properties file, the workflow `ConfigMap`, or environment variable setup.
+
+.Procedure
+
+. Use the following format to compose your property keys:
++
+[source,text]
+----
+quarkus.openapi-generator.[filename].auth.[security_scheme_name].[auth_property_name]
+----
++
+* `filename` is the sanitized name of the file containing the OpenAPI specification, such as security_example_json.
+* `security_scheme_name` is the sanitized name of the security scheme object definition in the OpenAPI specification file, such as http_basic_example or api_key_example.
+* `auth_property_name` is the name of the property to configure, such as username. This property depends on the defined security scheme type.
+
```

Comment:
[NOTE]
====
When you use environment variables, apply MicroProfile environment variable mapping rules and replace any non-alphabetic characters with underscores `_` as shown in the following examples.
====

Reply:
Added